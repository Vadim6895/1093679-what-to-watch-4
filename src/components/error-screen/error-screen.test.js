import React from "react";
import renderer from "react-test-renderer";
import ErrorScreen from "./error-screen.jsx";

const errorText = `Error 401 unauthorized`;

it(`Should ErrorScreen render correctly`, () => {
  const tree = renderer
  .create(
      <ErrorScreen
        error={errorText}
      />)
  .toJSON();

  expect(tree).toMatchSnapshot();
});
