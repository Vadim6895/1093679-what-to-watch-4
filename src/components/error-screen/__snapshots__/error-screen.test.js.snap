// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should ErrorScreen render correctly 1`] = `
<div
  style={
    Object {
      "background": "rgb(0, 0, 0)",
      "height": "100vh",
      "width": "100%",
    }
  }
>
  <div
    style={
      Object {
        "background": "rgb(0, 0, 0)",
        "color": "white",
        "fontSize": "30px",
        "margin": "0 auto",
        "paddingTop": "20px",
        "width": "600px",
      }
    }
  >
    Произошла ошибка при загрузке страницы
  </div>
  <p
    style={
      Object {
        "background": "rgb(0, 0, 0)",
        "color": "white",
        "margin": "0 auto",
        "paddingTop": "50px",
        "width": "500px",
      }
    }
  >
    Попробуйте позже...
  </p>
  <p
    style={
      Object {
        "background": "rgb(0, 0, 0)",
        "color": "white",
        "margin": "0 auto",
        "paddingTop": "50px",
        "width": "500px",
      }
    }
  >
    [object Object]
  </p>
</div>
`;
